packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -65,6 +65,14 @@ abstract class ResidentRunner {
       await cleanupAfterSignal();
       exit(0);
     });
+    ProcessSignal.SIGUSR1.watch().listen((ProcessSignal signal) async {
+      printStatus('Caught SIGUSR1');
```

Comment:
This should say what's going to happen next, e.g. "Caught SIGUSR1 - attempting hot reload".